src/interfaces/IGreeter.sol

```diff
@@ -72,4 +62,14 @@ interface IGreeter {
    * @return _balance  Current token balance of the caller
    */
   function greet() external view returns (string memory _greeting, uint256 _balance);
+
+  /*///////////////////////////////////////////////////////////////
+                            LOGIC
+  //////////////////////////////////////////////////////////////*/
```

Comment:
`greet()` should be under the `LOGIC` section as well, as it is an external function but not a variable.

Reply:
fixed